src/components/generic/movieModal/modalContent.tsx

```diff
@@ -25,19 +28,19 @@ export default ({
     imdb,
   },
 }) => {
-  {
-    open ? useLockBodyScroll('lock') : useLockBodyScroll('unlock')
-  }
+  // {
```

Comment:
I suggest to not leave comments. If it's required, I suggest adding a `// TODO: blabla...` or `// FIXME: blabla...` so it's easier to find the future.

Reply:
Gotcha.